src/Database/Redis/ManualCommands.hs

```diff
@@ -1114,17 +1114,30 @@ data XInfoConsumersResponse = XInfoConsumersResponse
     { xinfoConsumerName :: ByteString
     , xinfoConsumerNumPendingMessages :: Integer
     , xinfoConsumerIdleTime :: Integer
+    , xinfoConsumerInactive :: Maybe Integer -- since redis 7.0
```

Comment:
Ideally we want to add entries from Redis doc here. We can do that only in the fields we are adding:

e.g. Inactive:
> The number of milliseconds that have passed since the consumer's last successful interaction